src/tlo/scenario.py

```diff
@@ -393,9 +393,18 @@ def run_sample_by_number(self, output_directory, draw_number, sample_number):
         sample = self.get_sample(draw, sample_number)
         log_config = self.scenario.get_log_config(output_directory)
 
+        logger.info(key="message", data=f"Running draw {sample['draw_number']}, sample {sample['sample_number']}")
+
+        sim = Simulation(
+            start_date=self.scenario.start_date,
+            seed=sample["simulation_seed"],
+            log_config=log_config,
+            resourcefilepath=self.scenario.scenario_path)
+
```

Comment:
confused about the need for these changes? Should they be on this PR?!?!

Reply:
I was mostly following the Simulation objects to make changes to them wherever they are... If this change is not necessary for this PR, I can rollback

Reply:
Yeah if it's not necessary for the PR, then I think it's best to roll it back.